CMakeLists.txt

```
@@ -211,6 +211,11 @@ SET(INSTALL_PYTHON_FILES ON)        # To be removed when all applications are po
 include(install_function)
 include(KratosDependencies)
 
+# Logger configuration
+if(KRATOS_COLORED_LOGGING MATCHES ON)
```

if we change it globally then I would change the name
```suggestion
if(KRATOS_COLORED_OUTPUT MATCHES ON)
```

lets first hear what others think I suggest

---

done